server/services/tags_service.py

```diff
@@ -12,3 +20,20 @@ def get_all_organisation_tags():
     def get_all_campaign_tags():
         """ Get all org tags"""
         return Tags.get_all_campaigns()
+
+    @staticmethod
+    def delete_organisation_tags(organisation_tag):
+        """ Delete organisation tag"""
+        organisation_tags = Project.get_all_organisations_tag().tags
```

Comment:
If the purpose of the method is to delete duplicate tags it might be better to pass the tag ID rather than the name, but this is probably fine??